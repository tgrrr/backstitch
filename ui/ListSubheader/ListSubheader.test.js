import React from 'react'
import renderer from 'react-test-renderer'
import { ListSubheader } from '.'

it('ListSubheader: default', () => {
    const component = renderer.create(
        <ListSubheader>
            <>
                ListSubheader
            </>
        </ListSubheader>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
