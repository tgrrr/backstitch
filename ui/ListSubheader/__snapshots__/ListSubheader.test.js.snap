// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListSubheader: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Listsubheader "
    data-testid="Listsubheader"
  >
    <li
      className="MuiListSubheader-root MuiListSubheader-sticky MuiListSubheader-gutters"
    >
      ListSubheader
    </li>
  </div>
</div>
`;
